CMakeLists.txt

```diff
@@ -42,6 +43,19 @@ add_custom_target(coverage
   COMMENT "Generate code coverage stats"
 )
 
+if(DEFINED ENV{GITHUB_TOKEN})
+  set(COVERALLS_COMMAND ${LUACOVCOVERALLS} -v -r ${PROJECT_SOURCE_DIR} --repo-token $ENV{GITHUB_TOKEN})
+else()
+  set(COVERALLS_COMMAND ${CMAKE_COMMAND} -E echo "Skipped uploading to coveralls.io: no token.")
```

Comment:
Are we going to support uploading results when running tests locally?

If yes, there should be an explanation (GITHUB_TOKEN is generated [automatically](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token), and I can't do it locally). 

Otherwise, the message should be different (e.g. "Not in CI").